<?xml version="1.0"?>
<launch>
	<node name="netout_image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
		<remap from="image" to="yolo"/>
		<param name="autosize" value="true" />
		<param name="image_transport" value="compressed" />
	</node>

	<node name="netout_corr_image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
		<remap from="image" to="yolo_corr"/>
		<param name="autosize" value="true" />
		<param name="image_transport" value="compressed" />
	</node>

 </launch>
 